update-nixpkgs.py

```diff
@@ -111,8 +126,35 @@ def prefetch_nixpkgs(nixos_version: str) -> dict[str, str]:
     return prefetch_result
 
 
-def get_package_list():
-    run_on_hydra("")
+def update_package_list(local_path: Path):
+    basedir = "$XDG_RUNTIME_DIR"
+    subprocess.run(
+        ["rsync", "-a", local_path, f"hydra01:{basedir}"], check=True
+    )
+    dest = f"{basedir}/{local_path.name}/"
+    proc = run_on_hydra(
+        f"(cd {dest}; eval $(./dev-setup); set pipefail; nix-build ./get-package-versions.nix | xargs jq .)"
```

Comment:
Why do we need to use `jq` here if we re-format the json anyways in line 157 before writing it to file?

Reply:
yes, jq is not needed anymore, cat also works